package.json

```diff
@@ -86,7 +87,7 @@
     "prepublishOnly": "yarn -s build",
     "test": "jest --verbose --testTimeout 120000",
     "test:integration": "jest --verbose --testRegex '.+/integration/.+\\.spec\\.ts$'",
-    "test:unit": "jest --verbose --testRegex '.+/src/.+\\.spec\\.ts$'",
+    "test:unit": "node --experimental-worker node_modules/.bin/jest --verbose --testRegex '.+/src/.+\\.spec\\.ts$'",
```

Comment:
Required because now unit tests include a module that imports the worker module. This is not needed for system/e2e tests because they go through spawn which in turn runs through the CLI shebang, which has the flag there.